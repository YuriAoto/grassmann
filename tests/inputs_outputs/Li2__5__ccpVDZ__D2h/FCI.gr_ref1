Grassmann
Exploring the geometry of the electronic wave functions space
Yuri Aoto - 2018, 2019, 2020

Current git commit: d08f010b2b9fea125a521ca84f2602dc10a6d81e

Directory:
/home/yuriaoto/Documents/Codes/grassmann/tests/inputs_outputs/Li2__5__ccpVDZ__D2h

Command:
../../../Grassmann --method CCD_mani_minD --molpro_output FCI.out

Starting at 26 May 2021 - 20:53

it.   dist      |Z|       |J|        time in iteration
 0    0.32659  0.3255  0.3255   0:00:00.040915
 1    0.026399     0.0     0.0   0:00:00.056209
D(FCI, CCD manifold) = 0.02639859

Memory usage:
   Max. allocation: 5.28125 kB
   Max. used: 11.3359375 kB
Ending at 26 May 2021 - 20:53
Total time: 0:00:00.134586
