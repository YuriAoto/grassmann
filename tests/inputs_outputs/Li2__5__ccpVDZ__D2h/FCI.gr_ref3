Grassmann
Exploring the geometry of the electronic wave functions space
Yuri Aoto - 2018, 2019, 2020

Current git commit: d08f010b2b9fea125a521ca84f2602dc10a6d81e

Directory:
/home/yuriaoto/Documents/Codes/grassmann/tests/inputs_outputs/Li2__5__ccpVDZ__D2h

Command:
../../../Grassmann --method CCSD_mani_minD --molpro_output FCI.out --cc_diag_hess

Starting at 26 May 2021 - 20:57

it.   dist      |Z|       |J|        time in iteration
 0    0.32659  0.3266  0.3266   0:00:00.037221
 1    0.00034844  0.0003486  0.0003486   0:00:00.034186
 2    9.1984e-06  9.202e-06  9.205e-06   0:00:00.033888
 3    2.428e-07  2.43e-07  2.43e-07   0:00:00.034316
 4    6.4108e-09  6.415e-09  6.417e-09   0:00:00.035028
D(FCI, CCSD manifold) = 0.00000000

Memory usage:
   Max. allocation: 5.28125 kB
   Max. used: 11.4140625 kB
Ending at 26 May 2021 - 20:57
Total time: 0:00:00.213335
