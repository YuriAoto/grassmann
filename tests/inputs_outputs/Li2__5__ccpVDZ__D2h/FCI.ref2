Grassmann
Exploring the geometry of the electronic wave functions space
Yuri Aoto - 2018, 2019, 2020

Current git commit: d08f010b2b9fea125a521ca84f2602dc10a6d81e

Directory:
/home/yuriaoto/Documents/Codes/grassmann/tests/inputs_outputs/Li2__5__ccpVDZ__D2h

Command:
../../../Grassmann --method CCSD_mani_minD --molpro_output FCI.out

Starting at 26 May 2021 - 20:53

it.   dist      |Z|       |J|        time in iteration
 0    0.32659  0.3264  0.3266   0:00:00.056762
 1    0.0023606  0.002366  0.002355   0:00:00.080126
 2    2.7914e-06  2.8e-06  2.784e-06   0:00:00.080407
 3    1.1188e-12  1.122e-12  1.116e-12   0:00:00.080608
D(FCI, CCSD manifold) = 0.00000000

Memory usage:
   Max. allocation: 5.28125 kB
   Max. used: 11.4140625 kB
Ending at 26 May 2021 - 20:53
Total time: 0:00:00.354281
