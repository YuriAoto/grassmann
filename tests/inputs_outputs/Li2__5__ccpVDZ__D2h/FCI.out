
 Primary working directories    : /scr/yuriaoto/job.032980/molpro.4hncVih9Ui
 Secondary working directories  : /scr/yuriaoto/job.032980/molpro.4hncVih9Ui
 Wavefunction directory         : /scr/yuriaoto/job.032980/
 Main file repository           : /scr/yuriaoto/job.032980/molpro.4hncVih9Ui/

 id        : yuriaoto

 Nodes             nprocs
 t-rex.ufabc.int.br    1

 Using customized tuning parameters: mindgm=8; mindgv=16; mindgc=1; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=df  


 Variables initialized (933), CPU time= 0.00 sec
 ***,
 memory,40,m
 gprint,orbital,basis,civector
 gthresh,thrprint=0,printci=0.0
 file,3,wfu.dat,old
 
 {fci
 wf,6,1,0;
 }
 Commands  initialized (760), CPU time= 0.01 sec, 617 directives.
 Default parameters read. Elapsed time= 0.05 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2019.2 linked Abr  7 2019 15:10:35


 **********************************************************************************************************************************
 LABEL *                                                                                 
  64 bit serial version                                                                  DATE: 26-May-21          TIME: 20:44:22  
 **********************************************************************************************************************************

 SHA1:             3a8a58c585c929645742fb5a1b7541a4ad1da699
 **********************************************************************************************************************************


 THRESHOLDS:

 ZERO    =  1.00D-12  ONEINT  =  1.00D-12  TWOINT  =  1.00D-11  PREFAC  =  1.00D-14  XXXXXX  =  1.00D-09  EORDER  =  1.00D-04
 ENERGY  =  0.00D+00  ETEST   =  0.00D+00  EDENS   =  0.00D+00  THRDEDEF=  1.00D-06  GRADIENT=  1.00D-02  STEP    =  1.00D-03
 ORBITAL =  1.00D-05  CIVEC   =  1.00D-05  COEFF   =  1.00D-04  PRINTCI =  0.00D+00  PUNCHCI =  9.90D+01  OPTGRAD =  3.00D-04
 OPTENERG=  1.00D-06  OPTSTEP =  3.00D-04  THRGRAD =  1.00D-10  COMPRESS=  1.00D-11  VARMIN  =  1.00D-07  VARMAX  =  1.00D-03
 THRDOUB =  0.00D+00  THRDIV  =  1.00D-05  THRRED  =  1.00D-07  THRPSP  =  1.00D+00  THRDC   =  1.00D-10  THRCS   =  1.00D-10
 THRNRM  =  1.00D-08  THREQ   =  0.00D+00  THRDE   =  1.00D+00  THRREF  =  1.00D-05  SPARFAC =  1.00D+00  THRDLP  =  1.00D-07
 THRDIA  =  1.00D-10  THRDLS  =  1.00D-07  THRGPS  =  0.00D+00  THRKEX  =  0.00D+00  THRDIS  =  2.00D-01  THRVAR  =  1.00D-10
 THRLOC  =  1.00D-06  THRGAP  =  1.00D-06  THRLOCT = -1.00D+00  THRGAPT = -1.00D+00  THRORB  =  1.00D-06  THRMLTP =  0.00D+00
 THRCPQCI=  1.00D-10  KEXTA   =  0.00D+00  THRCOARS=  0.00D+00  SYMTOL  =  1.00D-06  GRADTOL =  1.00D-06  THROVL  =  1.00D-08
 THRORTH =  1.00D-08  GRID    =  1.00D-06  GRIDMAX =  1.00D-03  DTMAX   =  0.00D+00  THROCCDE=  1.00D-01  THROCCDE=  1.00D-01
 THRPRINT=  0.00D+00


 Variable memory set to   40000000 words,  buffer space 



 Permanent file  3  wfu.dat  assigned.  Implementation=df    Size= 14.09 MB


 PROGRAM * RESTART 


 Reading variables from file 3

 _FNUC           =      0.00000000
 _BASIS          =    CC-PVDZ
 _NELEC          =      6.00000000
 _PROGRAM        =    RHF-SCF
 _DMX_SCF        =      0.00000000
 _DMY_SCF        =      0.00000000
 _DMZ_SCF        =      0.00000000
 _HOMO           =      2.10000000
 _EHOMO          =     -0.18089003
 _LUMO           =      2.50000000
 _ELUMO          =      0.00860175
 _ENERGY         =    -14.86923112
 _ENERGY_METHOD  =    RHF
 _ENERGY_BASIS   =    cc-pVDZ
 _ENUC           =      1.80000000
 _EPS1(1:7)      =     -2.44754692    -0.18089003     0.04506451     0.14188695     0.17633377     0.31871808
                        0.45987531
 _EPS2(1:3)      =      0.03216991     0.16180289     0.35571558
 _EPS3(1:3)      =      0.03216991     0.16180289     0.35571558
 _EPS4           =      0.31871808
 _EPS5(1:7)      =     -2.44716908     0.00860175     0.08426831     0.25807888     0.31859242     0.39752665
                        0.94057166
 _EPS6(1:3)      =      0.06717723     0.21640604     0.51855068
 _EPS7(1:3)      =      0.06717723     0.21640604     0.51855068
 _EPS8           =      0.39752665
 _IPROC_MPP      =      0.00000000
 _IPROC_MPPX     =      0.00000000
 _ORBITAL        =   2100.30000000
 _STATUS         =      1.00000000
 _DATE           =    21-Oct-19
 _LASTORB        =    RHF
 _OUTPUT         =    /scr/yuriaoto/job.022402/RHF.xml
 _PGROUP         =    D2h 
 _TIME           =    10:20:37
 R               =      5.00000000

 Geometry written to block  1 of record 700


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL   RESTART
 CPU TIMES  *         0.12      0.01
 REAL TIME  *         0.16 SEC
 DISK USED  *        26.84 MB
 **********************************************************************************************************************************


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry LI     S cc-pVDZ              selected for orbital group  1
 Library entry LI     P cc-pVDZ              selected for orbital group  1
 Library entry LI     D cc-pVDZ              selected for orbital group  1


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Linear
 User-specified symmetry elements: X,Y,Z
 Symmetry elements: X,Y,Z
 Rotational constants:      20.8009354     20.8009354      0.0000000 GHz  (calculated with average atomic masses)

 Point group  D2h 



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  LI1     3.00    0.000000000    0.000000000   -2.500000000
   2  LI2     3.00    0.000000000    0.000000000    2.500000000

 Bond lengths in Bohr (Angstrom)

 1-2  5.000000000
     ( 2.645886055)


 BASIS DATA

   Nr Sym  Nuc  Type         Exponents   Contraction coefficients

    1.1 Ag    1  1s        1469.000000     0.000766   -0.000120    0.000000
    2.1 Ag    2  1s         220.500000     0.005892   -0.000923    0.000000
    3.1 Ag                   50.260000     0.029671   -0.004689    0.000000
                             14.240000     0.109180   -0.017682    0.000000
                              4.581000     0.282789   -0.048902    0.000000
                              1.580000     0.453123   -0.096009    0.000000
                              0.564000     0.274774   -0.136380    0.000000
                              0.073450     0.009751    0.575102    0.000000
                              0.028050    -0.003180    0.517661    1.000000
    4.1 Ag    1  2pz          1.534000     0.022784    0.000000
    5.1 Ag    2 -2pz          0.274900     0.139107    0.000000
                              0.073620     0.500375    0.000000
                              0.024030     0.508474    1.000000
    6.1 Ag    1  3d0          0.114400     1.000000
              2  3d0   
    7.1 Ag    1  3d2+         0.114400     1.000000
              2  3d2+  
    1.2 B3u   1  2px          1.534000     0.022784    0.000000
    2.2 B3u   2  2px          0.274900     0.139107    0.000000
                              0.073620     0.500375    0.000000
                              0.024030     0.508474    1.000000
    3.2 B3u   1  3d1+         0.114400     1.000000
              2 -3d1+  
    1.3 B2u   1  2py          1.534000     0.022784    0.000000
    2.3 B2u   2  2py          0.274900     0.139107    0.000000
                              0.073620     0.500375    0.000000
                              0.024030     0.508474    1.000000
    3.3 B2u   1  3d1-         0.114400     1.000000
              2 -3d1-  
    1.4 B1g   1  3d2-         0.114400     1.000000
              2  3d2-  
    1.5 B1u   1  1s        1469.000000     0.000766   -0.000120    0.000000
    2.5 B1u   2 -1s         220.500000     0.005892   -0.000923    0.000000
    3.5 B1u                  50.260000     0.029671   -0.004689    0.000000
                             14.240000     0.109180   -0.017682    0.000000
                              4.581000     0.282789   -0.048902    0.000000
                              1.580000     0.453123   -0.096009    0.000000
                              0.564000     0.274774   -0.136380    0.000000
                              0.073450     0.009751    0.575102    0.000000
                              0.028050    -0.003180    0.517661    1.000000
    4.5 B1u   1  2pz          1.534000     0.022784    0.000000
    5.5 B1u   2  2pz          0.274900     0.139107    0.000000
                              0.073620     0.500375    0.000000
                              0.024030     0.508474    1.000000
    6.5 B1u   1  3d0          0.114400     1.000000
              2 -3d0   
    7.5 B1u   1  3d2+         0.114400     1.000000
              2 -3d2+  
    1.6 B2g   1  2px          1.534000     0.022784    0.000000
    2.6 B2g   2 -2px          0.274900     0.139107    0.000000
                              0.073620     0.500375    0.000000
                              0.024030     0.508474    1.000000
    3.6 B2g   1  3d1+         0.114400     1.000000
              2  3d1+  
    1.7 B3g   1  2py          1.534000     0.022784    0.000000
    2.7 B3g   2 -2py          0.274900     0.139107    0.000000
                              0.073620     0.500375    0.000000
                              0.024030     0.508474    1.000000
    3.7 B3g   1  3d1-         0.114400     1.000000
              2  3d1-  
    1.8 Au    1  3d2-         0.114400     1.000000
              2 -3d2-  

 NUCLEAR CHARGE:                    6
 NUMBER OF PRIMITIVE AOS:          54
 NUMBER OF SYMMETRY AOS:           52
 NUMBER OF CONTRACTIONS:           28   (    7Ag  +    3B3u +    3B2u +    1B1g +    7B1u +    3B2g +    3B3g +    1Au  )
 NUMBER OF CORE ORBITALS:           2   (    1Ag  +    0B3u +    0B2u +    0B1g +    1B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF VALENCE ORBITALS:        8   (    2Ag  +    1B3u +    1B2u +    0B1g +    2B1u +    1B2g +    1B3g +    0Au  )


 NUCLEAR REPULSION ENERGY    1.80000000

 EXTRA SYMMETRY OF AOS IN SYMMETRY 1:   1 1 1 1 1 1 2
 EXTRA SYMMETRY OF AOS IN SYMMETRY 5:   1 1 1 1 1 1 2

 Eigenvalues of metric

         1 0.313E-01 0.101E+00 0.792E+00 0.106E+01 0.116E+01 0.124E+01 0.385E+01
         2 0.135E+00 0.138E+01 0.322E+01
         3 0.135E+00 0.138E+01 0.322E+01
         4 0.124E+01
         5 0.256E-02 0.901E-02 0.995E-01 0.761E+00 0.899E+00 0.115E+01 0.285E+01
         6 0.586E-01 0.230E+00 0.973E+00
         7 0.586E-01 0.230E+00 0.973E+00
         8 0.761E+00


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     0.262 MB (compressed) written to integral file (100.0%)


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:      13015.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:      13015      RECORD LENGTH: 524288

 Memory used in sort:       0.57 MW

 SORT1 READ       27440. AND WROTE       11779. INTEGRALS IN      1 RECORDS. CPU TIME:     0.01 SEC, REAL TIME:     0.00 SEC
 SORT2 READ       11779. AND WROTE       13015. INTEGRALS IN      1 RECORDS. CPU TIME:     0.01 SEC, REAL TIME:     0.01 SEC

 FILE SIZES:   FILE 1:    30.2 MBYTE,  FILE 4:     4.2 MBYTE,   TOTAL:     34.5 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       30.04       500      700      610      900      950      970     1000      129      960     1100   
                                          VAR     GEOM    BASINP   SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL       INT   RESTART
 CPU TIMES  *         0.27      0.15      0.01
 REAL TIME  *         0.40 SEC
 DISK USED  *        49.46 MB
 **********************************************************************************************************************************


 PROGRAM * FCI (Full CI)     Author: P.J. Knowles, 1984


 Molecular orbitals read from record     2100.3  Type=RHF/CANONICAL (state 1.1)

 Frozen orbitals:              2 (  1  0  0  0  1  0  0  0)
 Active orbitals:             26 (  6  3  3  1  6  3  3  1)
 Active electrons:             2
 Spin quantum number:          0.0
 Orbital pairs:                    68      42      42      30      55      42      42      30
 Strings:                           6       3       3       1       6       3       3       1
                                    6       3       3       1       6       3       3       1
 Determinants:                            110              84              84              60
                                          110              84              84              60

 Load integrals                1.0 sec
 Transform integrals           1.0 sec
 Storage for integrals:          17883
 Remaining memory:            39979181

 Core energy:             -14.27198869
 Run Hamiltonian processor DAVIDSON
 Symmetry:                   1
 Trial vector:                  0.00
 Result vector:                 0.00
 Maximum iterations:           90
 Convergence threshold:         0.0000100
 RHS vector:                    0.00
 Hamiltonian shift:             0.0000000
 Output threshold:              0.0000000
 Options:                        0
 Number of roots:             1

 Hamiltonian diagonaliser entered at time      1.06

 Initial configuration generated:
       1      1.0000000    -14.8692311

 It  Tr    CPU  Convergence   Energy       Pop     dE(next)

  1   1    1.1  1.00000000   -14.86923112
                                           0.29 -0.03441856 -0.03061654
  2   1    1.1  0.26482897   -14.89553166
                                           0.19 -0.00489208 -0.00432681
  3   1    1.1  0.09335474   -14.89982666
                                           0.26 -0.00066031 -0.00055756
  4   1    1.1  0.03465592   -14.90031287
                                           0.26 -0.00012553 -0.00010317
  5   1    1.1  0.01392912   -14.90041574
                                           0.28 -0.00002369 -0.00002085
  6   1    1.1  0.00586475   -14.90043509
                                           0.22 -0.00000368 -0.00000305
  7   1    1.1  0.00283848   -14.90043793
                                           0.28 -0.00000082 -0.00000076
  8   1    1.1  0.00124893   -14.90043869
                                           0.23 -0.00000009 -0.00000008
  9   1    1.1  0.00048590   -14.90043877
                                           0.31 -0.00000002 -0.00000002
 10   1    1.1  0.00019427   -14.90043879
                                           0.22 -0.00000000 -0.00000000
 11   1    1.1  0.00007471   -14.90043879
                                           0.40 -0.00000000 -0.00000000
 12   1    1.1  0.00003103   -14.90043879
                                           0.27 -0.00000000 -0.00000000
 13   1    1.1  0.00001492   -14.90043879
                                           0.27 -0.00000000 -0.00000000
 14   1    1.1  0.00000325   -14.90043879


RESULTS FOR STATE  1.1
=====================

 Correlation energy                    -0.031207667207
 !FCI STATE  1.1 Energy               -14.900438788920
 CI Vector (threshold=  0.00    )
       0.950589801393  1  2  3  1  2  3
       0.013250985023  1  2  3  1  2  4
      -0.009407558191  1  2  3  1  2  5
      -0.006608905930  1  2  3  1  2  6
       0.000000000000  1  2  3  1  2  7
      -0.002663039984  1  2  3  1  2  8
       0.013250985023  1  2  4  1  2  3
      -0.047206253591  1  2  4  1  2  4
       0.053800368699  1  2  4  1  2  5
      -0.007162340744  1  2  4  1  2  6
       0.000000000000  1  2  4  1  2  7
      -0.018271751952  1  2  4  1  2  8
      -0.009407558191  1  2  5  1  2  3
       0.053800368699  1  2  5  1  2  4
      -0.067749963811  1  2  5  1  2  5
       0.004439918289  1  2  5  1  2  6
       0.000000000000  1  2  5  1  2  7
       0.024384413282  1  2  5  1  2  8
      -0.006608905930  1  2  6  1  2  3
      -0.007162340744  1  2  6  1  2  4
       0.004439918289  1  2  6  1  2  5
      -0.004382451506  1  2  6  1  2  6
       0.000000000000  1  2  6  1  2  7
      -0.001789381914  1  2  6  1  2  8
       0.000000000000  1  2  7  1  2  3
       0.000000000000  1  2  7  1  2  4
       0.000000000000  1  2  7  1  2  5
       0.000000000000  1  2  7  1  2  6
      -0.005972855727  1  2  7  1  2  7
       0.000000000000  1  2  7  1  2  8
      -0.002663039984  1  2  8  1  2  3
      -0.018271751952  1  2  8  1  2  4
       0.024384413282  1  2  8  1  2  5
      -0.001789381914  1  2  8  1  2  6
       0.000000000000  1  2  8  1  2  7
      -0.011368203467  1  2  8  1  2  8
      -0.113379656359  1  2  9  1  2  9
      -0.080211663322  1  2  9  1  2 10
      -0.030475755399  1  2  9  1  2 11
      -0.080211663322  1  2 10  1  2  9
      -0.059154612950  1  2 10  1  2 10
      -0.024006644930  1  2 10  1  2 11
      -0.030475755399  1  2 11  1  2  9
      -0.024006644930  1  2 11  1  2 10
      -0.012809412750  1  2 11  1  2 11
      -0.113379656359  1  2 12  1  2 12
      -0.080211663322  1  2 12  1  2 13
      -0.030475755399  1  2 12  1  2 14
      -0.080211663322  1  2 13  1  2 12
      -0.059154612950  1  2 13  1  2 13
      -0.024006644930  1  2 13  1  2 14
      -0.030475755399  1  2 14  1  2 12
      -0.024006644930  1  2 14  1  2 13
      -0.012809412750  1  2 14  1  2 14
      -0.005972855727  1  2 15  1  2 15
      -0.072850524223  1  2 16  1  2 16
       0.052980794752  1  2 16  1  2 17
      -0.002916044031  1  2 16  1  2 18
      -0.029869627374  1  2 16  1  2 19
       0.000000000000  1  2 16  1  2 20
       0.006598028939  1  2 16  1  2 21
       0.052980794752  1  2 17  1  2 16
      -0.039958493862  1  2 17  1  2 17
       0.001637175222  1  2 17  1  2 18
       0.024113719567  1  2 17  1  2 19
       0.000000000000  1  2 17  1  2 20
      -0.006099740777  1  2 17  1  2 21
      -0.002916044031  1  2 18  1  2 16
       0.001637175222  1  2 18  1  2 17
      -0.000551234414  1  2 18  1  2 18
      -0.000210695006  1  2 18  1  2 19
       0.000000000000  1  2 18  1  2 20
      -0.000136782446  1  2 18  1  2 21
      -0.029869627374  1  2 19  1  2 16
       0.024113719567  1  2 19  1  2 17
      -0.000210695006  1  2 19  1  2 18
      -0.017407096325  1  2 19  1  2 19
       0.000000000000  1  2 19  1  2 20
       0.005697883069  1  2 19  1  2 21
       0.000000000000  1  2 20  1  2 16
       0.000000000000  1  2 20  1  2 17
       0.000000000000  1  2 20  1  2 18
       0.000000000000  1  2 20  1  2 19
      -0.001882994311  1  2 20  1  2 20
       0.000000000000  1  2 20  1  2 21
       0.006598028939  1  2 21  1  2 16
      -0.006099740777  1  2 21  1  2 17
      -0.000136782446  1  2 21  1  2 18
       0.005697883069  1  2 21  1  2 19
       0.000000000000  1  2 21  1  2 20
      -0.003411939145  1  2 21  1  2 21
      -0.000895780933  1  2 22  1  2 22
      -0.001846053603  1  2 22  1  2 23
      -0.001303618342  1  2 22  1  2 24
      -0.001846053603  1  2 23  1  2 22
      -0.003839249330  1  2 23  1  2 23
      -0.002566462693  1  2 23  1  2 24
      -0.001303618342  1  2 24  1  2 22
      -0.002566462693  1  2 24  1  2 23
      -0.002255919261  1  2 24  1  2 24
      -0.000895780933  1  2 25  1  2 25
      -0.001846053603  1  2 25  1  2 26
      -0.001303618342  1  2 25  1  2 27
      -0.001846053603  1  2 26  1  2 25
      -0.003839249330  1  2 26  1  2 26
      -0.002566462693  1  2 26  1  2 27
      -0.001303618342  1  2 27  1  2 25
      -0.002566462693  1  2 27  1  2 26
      -0.002255919261  1  2 27  1  2 27
      -0.001882994311  1  2 28  1  2 28
 /EOF
 !FCI STATE  1.1 Dipole moment          0.00000000     0.00000000     0.00000000
 Dipole moment /Debye                   0.00000000     0.00000000     0.00000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       30.04       500      700      610      900      950      970     1000      129      960     1100   
                                          VAR     GEOM    BASINP   SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL       FCI       INT   RESTART
 CPU TIMES  *         0.30      0.03      0.15      0.01
 REAL TIME  *         0.45 SEC
 DISK USED  *        49.46 MB
 **********************************************************************************************************************************
 Molecular orbitals read from record     2100.3  Type=RHF/CANONICAL (state 1.1)
 Occupation numbers read from record     2100.3  Type=RHF/RHF (state 1.1)
 Orbital energies read from record       2100.3  Type=RHF/CANONICAL (state 1.1)
 Redundancy group numbers read from rec  2100.3  Type=RHF/RHF (state 1.1)

 FCI/cc-pVDZ energy=    -14.900438788920

             FCI
    -14.90043879
 **********************************************************************************************************************************
 Molpro calculation terminated
